hieradata_aws/class/integration/router_backend.yaml

```diff
@@ -1,4 +1,37 @@
 govuk_env_sync::tasks:
+  "pull_router_daily":
+    ensure: "present"
```

Comment:
Just looked at the diff for when the sync was activated and spotted 'ensure' has now been set to 'present' from 'absent' - https://github.com/alphagov/govuk-puppet/blob/145bd8c1cbe7364b191907914050261c5bbe9860/hieradata_aws/class/integration/router_backend.yaml#L3, just wondered if this was intentional?

Reply:
Indeed it is intentional - we want to turn it back on ;-).

Reply:
Ohh okay lol, this is my lack of knowledge of puppty things shining through! 😆